### ISSUE TITLE
Feature request: Google Wallet support

### ISSUE THREAD

**Issue by obskyr** [itchio/itch.io#83]
PayPal has some less-than-desirable user agreement stipulations, and beyond that there aren't many user-friendly alternatives. I frequently find myself wishing to be able to use Google Wallet to pay, and I assume the same goes for receiving payment.

I've said this a few times on Twitter and such, but I guess it got lost. Hope you'll consider adding it!

**Comment by Xananax** [comment]
I know +1 are discouraged, but I'm going to allow myself to anyway: where I live, PayPal is not an option. That locks a part of itch.io's content from me, which drives me to buy the game elsewhere if it is available, or resolve to not be able to play the game at all.  

I want to stress that fact that absolutely *any* other payment gateway works worldwide; PayPal is the only one that blocks certain countries (and for no stated reason to boot). In other words, implementing absolutely anything else would allow a part of your audience to buy from Itch. A marginal increase, for sure, as this audience is in specific third-world countries, but a gain nonetheless.

**Comment by fasterthanlime** [comment]
> implementing absolutely anything else

we do support Stripe, but point taken!

**Comment by Xananax** [comment]
Sorry then!  
I just didn't see anything other than PayPal.  
I got told here: https://itch.io/t/20111/alternate-payment-methods  
That developers could choose PayPal or not on a case per case basis, I suppose no one chooses stripe?

**Comment by fasterthanlime** [comment]
> That developers could choose PayPal or not on a case per case basis, I suppose no one chooses stripe?

See https://itch.io/docs/creators/payments for an authoritative source on how payments work.

@leafo could give hard numbers on the numbers of sellers who chose direct payments *and* have a stripe account linked, but it's worth noticing that all sellers that are using the 'deferred payments' support both Paypal and Stripe (incl. bitcoin support)

**Comment by Xananax** [comment]
I see; in that case, I suppose it's not really something that can be fixed with a feature implementation. I would be grateful if you can have a note on the page where a dev chooses payment options, along the form of "if you choose PayPal, some countries won't be able to buy your game". Sadly, as [this page](http://smallbusiness.chron.com/country-doesnt-work-paypal-66099.html) says, PayPal doesn't maintain a list of countries it doesn't support, so information is scarce.  
But this is another conversation and probably doesn't have its place in a bug tracker and surely not in this particular feature request. Sorry for spamming it. Thanks for the prompt answer!

**Comment by Guard13007** [comment]
Sorry if this is too unrelated, but.. if I have direct payout selected, am I not accepting Bitcoin payments?

**Comment by fasterthanlime** [comment]
> Sorry if this is too unrelated, but.. if I have direct payout selected, am I not accepting Bitcoin payments?

No. Stripe's bitcoin support is 1) limited to USD 2) requires being enabled on a per-account basis (you have to apply for their beta and they have to approve it). For those reasons, bitcoin payments are only accepted using deferred payments at the moment.

**Comment by hawkerm** [comment]
+1

**Comment by obskyr** [comment]
*Google Wallet for digital goods* [is dead](https://support.google.com/payments/merchant/answer/6107573?hl=en&ref_topic=4490615), sadly. I have no idea of why they would do that, but... they did, nonchalantly giving the non-evil online payments market away to Stripe. They shut it down in March 2015, but honestly I'm still a bit mad about that one.

In addition to this small tragedy, the more recent Android Pay isn't an alternative - the Android Pay platform [also prohibits the sale of digital goods](https://developers.google.com/android-pay/faq#android_pay_questions). This means Google doesn't offer a platform for digital goods payments *at all* at the moment.

In the world of 2016, the alternatives that exist are the terrible PayPal, or the admittedly pretty fantastic Stripe - or any one of a collection of super obscure payment processors no one feels entirely comfortable using, I suppose. It's not really a problem since Stripe exists, I guess, but the lack of alternatives is still saddening.

**Comment by fasterthanlime** [comment]
> In the world of 2016, the alternatives that exist are the terrible PayPal, or the admittedly pretty fantastic Stripe - or any one of a collection of super obscure payment processors no one feels entirely comfortable using, I suppose. It's not really a problem since Stripe exists, I guess, but the lack of alternatives is still saddening.

That's more or less accurate unfortunately :( A bunch of payment processors have reached out to us over the past two years, but none of them have APIs comparable to Paypal or Stripe. They usually have better deals but are a lot more overhead for us, so we haven't really had a chance to give them a try yet.

**Comment by Xananax** [comment]
I'll allow myself to circle back on this because of all the games I'd like to buy and that I can't, because they use Paypal. I even have a [playlist](https://itch.io/c/134443/paypal-only-cant-buy).

I understand Stripe is _nominally available_, but I believe the easiness of clicking a button vs "contacting support" makes this option far, far less attractive, to the point of non-existence:

![image](https://user-images.githubusercontent.com/836082/57955000-415a3280-78fd-11e9-9089-9272529fc2e2.png)

"Stripe" isn't even mentioned, the copy makes it seems as if this is a technical problem to solve.

This might warrant its own issue (something like `make Stripe easier to set up` or `Change the copy of the payment page`), but in reason of the large number of more important things in the tracker, I surmised it was better to keep this co-located with the prior conversation, even if the title about Google Wallet isn't really relevant anymore. 

----

I will remind you of my comment above: Paypal blocks certain countries, does not disclose which, doesn't say that it does on any official channel, and refuses to comment or even acknowledge this.  

I can see four reasons for encouraging Stripe over Paypal:

1. Granted, the countries blocked don't have a lot of purchasing power, but collectively, we are a few million people. The addressable market isn't huge (a lot of those people wouldn't be interested in indie games), but not negligible either. We are also incredibly non-vocal due to years of shrugs from the online community and total silence from Paypal, so lack of complaints does not mean that there isn't an opportunity cost (though I will acknowledge that this probably falls in the "we're dozens!" region).
2. There is also the ethical aspect, specially in relation to the branding of Itch as a platform that promotes transparency, empathy, and progressive ideals. On top of the - already pretty iffy - practice of blacklisting countries for no good reason, Paypal has a history of supporting less than stellar individuals and groups (though I will also acknowledge they're doing better lately)
3. Please (T-T)
4. I am determined to not touching Steam or Epic with a ten foot pole, but it's frustrating when I need to convince Indies to "please get on Itch so I can buy your game there. Also can I ask you to not use Paypal because...". It's difficult to fit in a tweet too.
5. Please!
